downstream/modules/platform/proc-pulling-the-secret.adoc

```diff
@@ -0,0 +1,32 @@
+[id="proc-pulling-the-secret"]
+
+= Pulling the secret for OpenShift Container Platform deployments
+
+[NOTE]
+====
+This does not apply to {PlatformNameShort} on {Azure}.
+====
+
+If you are using the default {ExecEnvShort} provided with {ControllerName} to run on remote execution nodes, you must add a pull secret in {ControllerName} that contains the credential for pulling the {ExecEnvShort} image.
+
+To do this, create a pull secret on the {ControllerName} namespace and configure the `ee_pull_credentials_secret parameter in the Operator as follows:
```

Comment:
I think the closing quote is missing here: configure the `ee_pull_credentials_secret parameter in